### PR TITLE
Initial addition of the survey directory/module

### PR DESCRIPTION
**Pull request by MikeDeberg**

### CODE REVIEW

File: yax/modules/read_prep/read_prep.py [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+#
+# optional
+#
+# [adapters]
+# adapter_list = /path/to/adapter.list
+# adapter_tolerance = 10
+#
+# [quality_filter]
+# minimum_quality = 25
+# max_below_threshold = 6
```
Comment (ebolyen) [inline yax/modules/read_prep/read_prep.py:36]: Remove or make a docstring

File: yax/modules/read_prep/read_prep.py [hunk 2]
```diff
@@ -0,0 +1,85 @@
+import configparser
+import state.type.artifact as artifact
```
Comment (ebolyen) [inline yax/modules/read_prep/read_prep.py:2]: wrong imports

File: yax/modules/taxa_identification/GetTaxIDsFromAlignment.py [hunk 3]
```diff
@@ -0,0 +1,64 @@
+def analyze_sam_file(sam_fp, nm_max, taxids_of_interest):
+    ###########################################################################
```
Comment (ebolyen) [inline yax/modules/taxa_identification/GetTaxIDsFromAlignment.py:2]: make a docstring

**Comment by MikeDeberg** [comment]
Additional files added to pull requests are inappropriate. Closing request

**Pull request closed without merging**